src/components/PlaylistForm.vue

```diff
@@ -40,7 +40,12 @@
             </VBtn>
           </VCol>
         </VRow>
-        <VRow no-gutters align="center" justify="center">
+        <VRow
+          no-gutters
+          align="center"
+          justify="center"
+          v-if="newPlaylist.item_ids.length"
```

Comment:
Will this ever be the case?

Reply:
This form is used for both new playlists and editing, so there should be plenty of cases were `newPlaylist` contains items

Reply:
Hmm, `newPlaylist` doesn't seem like the best name to me in that case.

Reply:
I don't see the problem + it is the convention we have in all other forms that are used for both creating and updating